source/index.txt

```diff
@@ -64,8 +64,8 @@ API
 ---
 
 For detailed information about classes and methods in the MongoDB
-Node.js driver, see the :node-api-3.6:`MongoDB Node.js driver API
-documentation <>`.
+Node.js driver, see the `MongoDB Node.js driver API documentation
+<{+api+}>`__ .
```

Comment:
Question: The :node-api-3.6: constant is seems to still be used a good bit throughout the docs. It seems like it could cause issues to have some links use :node-api-3.6: and some links use the constant, but I could be incorrect. Should all :node-api-3.6: links be updated? If not, would it make sense to make another ticket to perform this replacement?

I read the A/C for the ticket and it seemed sparse (did not include TOC update), so not sure if this should or should not be a part of this ticket.

Reply:
Good catch, there are many more uses of :node-api-3.6: and I do think we should keep it consistent.

I tested a handful of the :node-api-3.6: links and they all work, which raises the YAGNI-esque question for me - why replace these with a new api constant?

I reverted the change on the index page back to using :node-api-3.6: - this seems like the simplest solution. I'll bring it up in standup and please let me know what you think.